tests/integration_cases/test_raft.rs

```diff
@@ -4049,3 +4049,65 @@ fn test_prevote_with_check_quorum() {
     assert_eq!(network.peers[&2].state, StateRole::Leader, "peer 2 state",);
     assert_eq!(network.peers[&3].state, StateRole::Follower, "peer 3 state",);
 }
+
+/// Tests if unapplied conf change is checked before campaign.
+#[test]
+fn test_conf_change_check_before_campaign() {
```

Comment:
if we don't call `self.hub(true)`, the test can fail?

Reply:
Yes. The case was written before the fix, and it failed.